RELEASE.md

```diff
@@ -17,6 +17,7 @@
 * Added support for generator functions as nodes, i.e. using `yield` instead of return.
   * Enable chunk-wise processing in nodes with generator functions.
   * Save node outputs after every `yield` before proceeding with next chunk.
+* Added support for Azure Data Lake Storage Gen2 URIs in datasets.
```

Comment:
Isn't this more of a bug fix?

Reply:
Yeah that's fair, I think it's a bit of both. Changed the release note in [a33664c](https://github.com/kedro-org/kedro/pull/2190/commits/a33664cd54630b8c7adcdd28a0a3c412fab7a672).